docs/src/inputs/input/devSandbox/devSandbox.jsx

```diff
@@ -0,0 +1,565 @@
+                    >
+                        Auto Focus
+                    </Heading>
+
+                    <Typography
+                        variant="body1"
+                    >
+                        Allows the Input to receive focus on load.
+                    </Typography>
+                </MarkdownContainer>
+
+                <Example
+                    rawCode={require('!!raw-loader!../examples/exampleInputAutoFocus').default}
+                >
+                    <ExampleInputAutoFocus />
+                </Example>
+
+                <MarkdownContainer>
+                    <Heading
+                        anchorLink="disabled"
+                        variant="h2"
+                    >
+                        Disable
+                    </Heading>
+
+                    <Typography
+                        variant="body1"
+                    >
+                        Indicates that the input is not available for interaction.
+                    </Typography>
+                </MarkdownContainer>
+
+                <Example
+                    rawCode={require('!!raw-loader!../examples/exampleInputDisabled').default}
+                >
+                    <ExampleInputDisabled />
+                </Example>
+
+                <MarkdownContainer>
+                    <Heading
+                        anchorLink="disabled"
+                        variant="h2"
+                    >
+                        Error
+                    </Heading>
+
+                    <Typography
+                        variant="body1"
+                    >
+                        Indicates that the input has an error by passing an boolean value.
+                    </Typography>
+                </MarkdownContainer>
+
+                <Example
+                    rawCode={require('!!raw-loader!../examples/exampleInputError').default}
+                >
+                    <ExampleInputError />
+                </Example>
+
+                <MarkdownContainer>
+                    <Heading
+                        anchorLink="disabled"
+                        variant="h2"
+                    >
+                        Error with custom message
+                    </Heading>
+
+                    <Typography
+                        variant="body1"
+                    >
+                        Indicates that the input has an error by passing an string with a message.
+                    </Typography>
+                </MarkdownContainer>
+
+                <Example
+                    rawCode={require('!!raw-loader!../examples/exampleInputErrorWithText').default}
+                >
+                    <ExampleInputErrorWithText />
+                </Example>
+
+                <MarkdownContainer>
+                    <Heading
+                        anchorLink="disabled"
+                        variant="h2"
+                    >
+                        Fluid
+                    </Heading>
```

Comment:
`anchorLink` should be something like `"fluid"`

Reply:
Fixed. Please check this [Commit](https://github.com/saddlebackdev/react-cm-ui/pull/353/commits/46c752aa0e781aedba48ebab9e7a168431118aa2)

Reply:
Looks good.  Resolving.